plaid-stl/src/github/mod.rs

```diff
@@ -9,6 +9,25 @@ pub enum ReviewPatAction {
     Deny,
 }
 
+#[derive(Debug, Deserialize)]
+/// Set of user permissions, as returned by GitHub's API
+struct Permission {
+    pull: bool,
+    triage: bool,
+    push: bool,
+    maintain: bool,
+    admin: bool,
+}
+
+#[derive(Debug, Deserialize)]
+/// A collaborator on a GitHub repository
+pub struct RepositoryCollaborator {
+    login: String,
+    r#type: String,
```

Comment:
Generally I prefer `type_` and when used with serde you can use a rename directive. That just means that we prevent having `r#` throughout the codebase.

Also I think these variables need to be marked `pub` so they can be accessed by calling rules.